Makefile

```diff
@@ -277,6 +276,28 @@ build-%:
 		--platform linux/amd64,linux/arm64 \
 		.
 
+.PHONY: build-artemis
+build-artemis:
+	docker buildx build \
+		-f v2/artemis/Dockerfile \
+		-t $(DOCKER_IMAGE_PREFIX)artemis:$(IMMUTABLE_DOCKER_TAG) \
+		-t $(DOCKER_IMAGE_PREFIX)artemis:$(MUTABLE_DOCKER_TAG) \
+		--build-arg VERSION=$(VERSION) \
+		--build-arg COMMIT=$(GIT_VERSION) \
+		--platform linux/amd64,linux/arm64v8 \
+		.
+
+.PHONY: build-worker
+build-worker:
+	docker buildx build \
+		-f v2/worker/Dockerfile \
+		-t $(DOCKER_IMAGE_PREFIX)worker:$(IMMUTABLE_DOCKER_TAG) \
+		-t $(DOCKER_IMAGE_PREFIX)worker:$(MUTABLE_DOCKER_TAG) \
+		--build-arg VERSION=$(VERSION) \
+		--build-arg COMMIT=$(GIT_VERSION) \
+		--platform linux/amd64,linux/arm64v8 \
+		.
```

Comment:
These need different values for the `--platform` flag than other images do. The reason is that multiarch images don't exist for Alpine that can handle both `linux/amd64` and `linux/arm64`. The Dockerfiles in these cases select _different_ base images depending on the target architecture. They select either `amd64/___:___` or `arm64v8/___:___`. `arm64/___:___` images do not exist, so that's what forces us to use different values for the `--platform` flag.

`arm64` and `arm64v8` are the _same_ architecture afaik, and are in fact two different ways of saying what is _officially_ `aarch64`.

None of this is about an architectural mismatch. It's simply about strings. :sigh: